eng/scripts/Create-Apiview-Token-TypeSpec.ps1

```diff
@@ -0,0 +1,110 @@
+[CmdletBinding()]
+param (
+    [ValidateNotNullOrEmpty()]
+    [string] $Reviews,
+    [ValidateNotNullOrEmpty()]
+    [string] $WorkingDir,
+    [ValidateNotNullOrEmpty()]
+    [string] $OutputDir,
+    [string] $GitPat = ""
+)
+
+Set-StrictMode -Version 3

```

Comment:
Doesn't have to be now but can you file an issue to figure out how to use https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/TypeSpec-Project-Sync.ps1 so we can share this syncing process with the other emitters?

Reply:
Done. https://github.com/Azure/azure-sdk-tools/issues/5856